modules/ROOT/pages/design-named-examples.adoc

```diff
@@ -1,44 +1,40 @@
-= Guide to Named Examples in RAML 1.0
+= Guide to Defining Examples in RAML 1.0
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
 
-If you want to use one or more named examples for a type, trait, or resource type in your RAML 1.0 API specification, you must follow these three rules:
+When defining examples in external files for a type, trait, or resource type in your RAML 1.0 API specification, you should follow these rules:
```

Comment:
@kbostrom , _sigh_... I wonder what their argument is. Something like this?

This use of "you" and the use of modal verbs are a type of weed in our topics.
We should devote ourselves to uprooting every single weed that appears in our documentation.
  Because uprooting weeds is a greater priority than pruning, watering, and otherwise tending to our strengths,
  And because every type weed is as pestilent as all of the other types.
Therefore, we should continue to divert ourselves from tending our strengths and instead deracinate this pestilent weed.